.travis.yml

```diff
@@ -4,6 +4,9 @@
 language: objective-c
 
 env:
+  matrix:
+    
+    - CONDA_PERL=5.20.3.1
```

Comment:
It is the same deal as Python. I do agree with you though @ocefpaf. I'm also concerned about the way it has been done in conda-build - I'd expect most applications to use perl as an executable, rather than an `.so`, so we may be better off doing a `perl *` dependency and not pinning it at all...